CONTRIBUTING.md

```diff
@@ -4,6 +4,25 @@ You are most welcome to create pull requests and issues to ecChronos.
 Before creating pull requests it is recommended to first discuss your idea with at least one of the owners of the repository.
 For keeping track of the history it is recommended that most communication is performed or summarized within pull requests or issues.
 
+## Branches
+
+Target the minimum possible branch version when fixing an issue.
```

Comment:
Hmm.. I'm thinking that if it is a bug fix then we should target ecchronos-1.0 when applicable. But if it's a new feature then we should target the master branch, etc.

Maybe a better way to phrase it would be:

> Target the lowest applicable version of ecChronos when fixing an issue.
> Bug fixes should be targeted to the lowest maintained version where the bug resides.
> New features should, in general, be added to the master branch.

Reply:
I think that would explain things better :-)